rest-api/src/main/java/gov/cms/qpp/conversion/api/services/CpcFileServiceImpl.java

```diff
@@ -29,6 +37,24 @@
 		return transformMetaDataToUnprocessedCpcFileData(metadata);
 	}
 
+	/**
+	 * Retrieves the file location id and retrieves the file if it is an unprocessed cpc+ file
+	 *
+	 * @param fileId {@link Metadata} identifier
+	 * @return file contents as a {@link String}
+	 * @throws IOException
+	 */
+	public String getFileById(String fileId) throws IOException, NoFileInDatabaseException {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the declaration of thrown exception 'gov.cms.qpp.conversion.api.exceptions.NoFileInDatabaseException' which is a runtime exception. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)